src/styles/loader.scss

```diff
@@ -0,0 +1,24 @@
+.bar {
+    width: 15px;
+    height: 6px;
+    border-radius: 2px;
+    background-color: $loading-bar-background;
+}
+
+.loader
```

Comment:
Possible error condition: did you mean `.loader .bar` or `.loader, .bar`? If the former, reduce this to one line; if the latter, you're missing the comma.